postgresql/Makefile

```diff
@@ -0,0 +1,10 @@
+BITNAMI_POSTGRES_RELEASE=13.2.24
+
+.PHONY: gen-energy-platform-postgresql
+gen-postgresql:
+	docker run -ti --rm \
```

Comment:
What do you think about adding namespace here, so that it don't have to be further patched?

Reply:
Happy for you to make a PR to make any changes you like after this is merged. Unless this is a blocker to getting it merged I don't want to add functionality that I don't need.

Reply:
Your team have around 200 Postgres instances, right? So, you don't need the namespace defined once, and you prefer to do it per instance?

Reply:
The expectation was to use kustomize to patch the namespace. Will use our own repo for now and show the working examples before we try and generalize.